routes/index.js

```diff
@@ -270,9 +327,14 @@ router.get('/story', function(req, res, next){
     });
 });
 
+/* GET new job page */
+router.get('/post_job', function(req, res, next) {
+  res.render('post_job', buildCreateJobMessage(req, "jobs"));
+});
+
 /* GET admin discover new page */
-router.get('/admin_discover_new', function(req, res, next) {
-  res.render('admin_discover_new', buildDefaultMessage(req, "about"));
+router.get('/post_story', function(req, res, next) {
```

Comment:
Should ensure that only admins can access these (auth.ensureUserIsAdmin or something like that)